app/graphql/context.js

```diff
@@ -31,7 +31,8 @@ export const extractOrgIdFromDefraIdToken = (sbi, token) => {
 export async function context({ request }) {
   const client = new MongoClient(config.get('mongo.mongoUrl'), {
     retryWrites: config.get('mongo.mongoOptions.retryWrites'),
-    readPreference: config.get('mongo.mongoOptions.readPreference')
+    readPreference: config.get('mongo.mongoOptions.readPreference'),
```

Comment:
Should the `MongoClient` be created in the `context` function? This means a new client will be created for every request to the DAL.